src/ngircd/irc-info.c

```diff
@@ -1514,10 +1515,13 @@ IRC_Send_NAMES(CLIENT * Client, CHANNEL * Chan)
 		return CONNECTED;
 
 	/* Secret channel? */
-	if (!is_member && Channel_HasMode(Chan, 's'))
+	bool secret_channel = Channel_HasMode(Chan, 's');
```

Comment:
Not 100% sure about very old compilers, probably this is supported "ever since", but please move the variable declaration at the top at the function and only do the assignment here.

Reply:
there's already `bool secret_channel = Channel_HasMode(Chan, 's');` a four lines before, so this should be fine

Reply:
hehe, no, I'm not convinced: it was your first commit which _introduces_ the `secret_channel` variable altogether 😎

Reply:
Oops, sorry